pagopa/pn-frontend-e2e
# PN-9433-e-2-e-destinatario-sezione-stato-piattaforma-disservizi-risolti

## Short description
There isn't a test related to the **PN-9433** as you can see in the [confluence page](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868811083)


## List of changes proposed in this pull request
- added new feature file `053_A1_visualizzazioneDisserviziRisoltiPF`
- refactored `handleDisservizio()` and `checkMonitoraggio()` (now `waitLoadServiceTable()`) method into `HelpdeskPage`
- added into `HelpdeskPage` new method  `checkServiceStatus() ` 
- added 2 enums: `Disservice `and `Status` in order to use them into `HelpdeskPage`
- fixed `getDigitalAddress()` of #138  where the base path was different 

## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_PF_VisualizzaDisserviziRisolti` and verify if the test works fine.
